docs/config-spec.md

```diff
@@ -17,73 +17,68 @@ For example:
 # Metadata
 #####################
 
-# TBD
-# Unique identifier for this observability pack
-# uuid: guid, required
+# A unique Observability Pack identifier
+uuid: guid, required
+
+# The name of the pack displayed everywhere the pack is referenced
+title: string, optional
```

Comment:
Should this be required? We will be using it to display the pack in the both catalogs.

Reply:
I would say yes to being required for the title field, you make a good point.  it's currently set to nullable in graphql 

<img width="347" alt="Screen Shot 2021-08-19 at 8 49 16 AM" src="https://user-images.githubusercontent.com/1395158/130090280-944ae1f2-2e2a-424b-8d7f-aa473943951b.png">